<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Rhythm_GameClass</class>
 <widget class="QMainWindow" name="Rhythm_GameClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rhythm_Game</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="start_button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>275</width>
      <height>54</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(C:\image/startButtonBasic.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="camera">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="exit_button">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>400</y>
      <width>275</width>
      <height>54</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(C:\image/quitButtonBasic.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="music_image">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>200</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(C:\image/music_image_1.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="upButtonBasic">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>20</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(C:\image/leftButtonBasic.png) 3 10 3 10;
border-top: 3px transparent;
border-bottom: 3px transparent; 
border-right: 10px transparent;
border-left: 10px transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="downButtonBasic">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>400</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(C:\image/rightButtonBasic.png) 3 10 3 10;
border-top: 3px transparent;
border-bottom: 3px transparent; 
border-right: 10px transparent;
border-left: 10px transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>camera</zorder>
   <zorder>exit_button</zorder>
   <zorder>start_button</zorder>
   <zorder>upButtonBasic</zorder>
   <zorder>downButtonBasic</zorder>
   <zorder>music_image</zorder>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="rhythm_game.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>start_button</sender>
   <signal>clicked()</signal>
   <receiver>Rhythm_GameClass</receiver>
   <slot>start_button()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>434</y>
    </hint>
    <hint type="destinationlabel">
     <x>143</x>
     <y>554</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exit_button</sender>
   <signal>clicked()</signal>
   <receiver>Rhythm_GameClass</receiver>
   <slot>exit_button()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>360</x>
     <y>434</y>
    </hint>
    <hint type="destinationlabel">
     <x>294</x>
     <y>578</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>upButtonBasic</sender>
   <signal>clicked()</signal>
   <receiver>Rhythm_GameClass</receiver>
   <slot>change_music()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>608</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>654</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>downButtonBasic</sender>
   <signal>clicked()</signal>
   <receiver>Rhythm_GameClass</receiver>
   <slot>change_music()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>614</x>
     <y>442</y>
    </hint>
    <hint type="destinationlabel">
     <x>654</x>
     <y>431</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>music_image</sender>
   <signal>clicked()</signal>
   <receiver>Rhythm_GameClass</receiver>
   <slot>chose_music()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>611</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>660</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>start_button()</slot>
  <slot>exit_button()</slot>
  <slot>change_music()</slot>
  <slot>chose_music()</slot>
 </slots>
</ui>
